src/status_im/chat/events/input.cljs

```diff
@@ -246,15 +245,18 @@
 
 (defn set-contact-as-command-argument
   "Sets contact as command argument for active chat"
-  [db {:keys [bot-db-key contact arg-index]}]
+  [{:keys [current-chat-id] :as db} {:keys [bot-db-key contact arg-index]}]
   (let [name    (str/replace (:name contact) (re-pattern const/arg-wrapping-char) "")
-        contact (select-keys contact [:address :whisper-identity :name :photo-path :dapp?])]
-    (-> (set-command-argument db arg-index name true)
-        (as-> fx
-            (merge fx (bots-events/set-in-bot-db
-                       (:db fx)
-                       {:path [:public (keyword bot-db-key)]
-                        :value contact})))
+        contact (select-keys contact [:address :whisper-identity :name :photo-path :dapp?])
+        command-owner (-> db
+                          (input-model/selected-chat-command current-chat-id)
+                          :command
```

Comment:
Looks like a code duplication, worth extracting as a separate function.

Reply:
I didn't want to do that now, as I plan to refactor the command loading as a next step, so the shape in db will be little bit different, then yes, I will extract that into separate function.